contracts/cm-v1.yaml

```diff
@@ -1654,4 +1712,37 @@ components:
                   enum: [HIU, HIP]
                 id:
                   type: string
-                  example: 100005
\ No newline at end of file
+                  example: 100005
+    HipInitiatedLinkRequest:
+      type: object
+      properties:
+        requestId:
+          type: string
+          format: uuid
+        timestamp:
+          $ref: '#/components/schemas/TimestampSchema'
+        cmId:
```

Comment:
do we want the "cmId" to be outside? now you have two attribute roots, cmId and patient. can we think of bringing under one, like majority or all of our apis are.

Reply:
you can bring the "cmid" inside the "patient_link" root attribute itself. like
<pre><code>
patient : {
  id: hinapatel@ncg, 
  hipId:  10000005, 
  referenceNumber: NCG1001,
  display: "Patient 1001 at TMH long distance care program"
  
  ...
} 
</code></pre>

Reply:
done

Reply:
I am wondering if the above is better or below
<pre> <code>
link: {
  hipid: 1000005,
  cmId: "hinapatel@ncg",
  patient: {
    "referenceNumber": "string",
    "display": "string",
    "careContexts": [
      {
        "referenceNumber": "string",
        "display": "string"
      }
    ]
  }
}
</code></pre>

@ganesanarun